lectures/week_07/README.md

```diff
@@ -0,0 +1,108 @@
+
+Now that we are running a Docker container, Docker, by default, will log you in as the "root" user. We need to run as the ubuntu user to match the RNAseq course tutorials.
+
+1. Switch User `su` to the ubuntu user:
+
+```bash
+su ubuntu
+```
+
+2. Source the pre-installed `.bashrc` file to configure your environment to match the RNAseq course:
+
+```bash
+source ~/.bashrc
+```
+
+NOTE: Using Docker and the persistent "workspace" volume we attached will allow you to start/stop as you wish. EVERYTIME YOU LOGIN TO THE DOCKER CONTAINER, YOU MUST LOGIN AS THE `ubuntu` USER *AND* `source ~/.bashrc` UPON EACH LOGIN.
+
+### Environment Setup
+
+Create a working directory and set the ‘RNA_HOME’ environment variable
+```
+mkdir -p ~/workspace/rnaseq/
+
+export RNA_HOME=~/workspace/rnaseq
+```
+
+Make sure whatever the working dir is, that it is set and is valid
+```
+echo $RNA_HOME
+```
+
+Since all the environment variables we set up for the RNA-seq workshop start with ‘RNA’ we can easily view them all by combined use of the env and grep commands as shown below. The env command shows all environment variables currently defined and the grep command identifies string matches.
+```
+env | grep RNA
+```
+
+In order to view the contents of this file, you can type:
+```
+less ~/.bashrc
+```
+To exit `less`, type `q`.
+
+### Known Issues/ Discrepancies from RNAbio website
+1. When running the check strandedness tool in the Module 1, RNAseq Data section, the docker run command cannot be run from within your `griffithlab/rnabio:0.0.1` docker session. To run it, we suggest that you open a new terminal window, `cd` into the `rnaseq` directory you created at the beginning of this assignment, and use the following command instead:
+```
+docker run -v $PWD/:/docker_workspace mgibio/checkstrandedness:latest check_strandedness --gtf /docker_workspace/refs/chr22_with_ERCC92_tidy.gtf --transcripts /docker_workspace/refs/chr22_ERCC92_transcripts.clean.fa --reads_1 /docker_workspace/data/HBR_Rep1_ERCC-Mix2_Build37-ErccTranscripts-chr22.read1.fastq.gz --reads_2 /docker_workspace/data/HBR_Rep1_ERCC-Mix2_Build37-ErccTranscripts-chr22.read2.fastq.gz
+```
+This is the same command as what is mentioned in the course webpage, except that instead of mounting (`-v` flag) `/home/ubuntu/workspace/rnaseq` to the docker image- which is where the data was stored for students running through the course on an AMI, you will instead mount whatever your current directory is. Also, this is different from an interactive session where we are able to enter the docker and run commands within it. Instead we are executing our command directly all in that one line of code.
+
+2. In various parts of RNAbio, in order to view HTML files, plots, etc., the tutorial suggests going to a public IPV4 address link in your browser window. That is only needed for the AMI. Since you'll be running everything locally, you can just find the files in your Finder window or File Explorer and open them directly!
```

Comment:
Suggested change:

Since you'll be running everything locally, you can use `open file.html` on mac, or `explorer.exe file.html` on Windows/WSL2 to open the file in your default browser.

Reply:
Incorporated this change in the new readme, thanks!